CIP-0PRS/CIP-0PRS.lagda.md

```diff
@@ -0,0 +1,1355 @@
+---
+Title: Peras - Faster Settlement on Cardano
```

Comment:
```suggestion
Title: Ouroboros Peras - Faster Settlement
```
PR title (this) is better of the two especially because the word `Cardano` is always redundant in a CIP.